### PR TITLE
@alloy <= Use Frameworks

### PR DESCRIPTION
**Pull request by orta**
I know what you're thinking, "but there's already an unmergable PR for frameworks." Well guess what, there's more. A bug in the de-duplicating version of means that you can't get away with `#import "Pods.h"` which meant that I had to ensure we didn't use any of these in our app ( which is what we should be doing anyway ) so I made this PR to give someone a roadmap for when frameworks gets in. 

~~This PR totally breaks Facebook login, we really need that auth lib finished~~
This has been re-implmented with the new Facebook API, so it will need some manual testing.

### CODE REVIEW

**Comment by alloy** [comment]
Yay, more :tada: :smile: 

> A bug in the de-duplicating version of means that you can't get away with `#import "Pods.h"` which meant that I had to ensure we didn't use any of these in our app ( which is what we should be doing anyway )

I’m not sure what this means. Is that a bug in CocoaPods? And what is it that we should have been using and what we shouldn’t have been using?

**Comment by orta** [comment]
CocoaPods allows accessing Pods' header files individually like `#import "HeaderInPods.h"`, which is wrong, it should be `#import <Pod/HeaderInPod.h>`. This PR is using [marius' PR](https://github.com/CocoaPods/CocoaPods/issues/3550) which briefly broke this "feature".

**Comment by alloy** [comment]
Gotcha, thanks :+1:

File: Artsy/Classes/ARAppDelegate.m [hunk 1]
```diff
@@ -344,33 +348,6 @@ - (void)showQuicksilver
     [navigationController pushViewController:adminSettings animated:YES];
 }
 
-- (void)checkForiOS7Deprecation
```
Comment (orta) [inline Artsy/Classes/ARAppDelegate.m]: compiling for iOS8 now, in theory we could do something similar for ios8 later next year

File: Podfile [hunk 2]
```diff
@@ -25,84 +29,92 @@ plugin 'cocoapods-keys', {
...
-pod 'Artsy+UILabels', :head
-pod 'Artsy+UIColors'
-pod 'Artsy-UIButtons', '~> 1.4'
-pod 'UIView+BooleanAnimations'
-
-if %w(orta ash artsy laura eloy sarahscott).include?(ENV['USER']) || ENV['CI'] == 'true'
-  pod 'Artsy+UIFonts', '1.0.0'
-else
-  pod 'Artsy+OSSUIFonts'
+target 'Artsy' do
```
Comment (orta) [inline Podfile]: moved our pods into the app target instead of to all targets :+1:

File: Podfile [hunk 3]
```diff
@@ -25,84 +29,92 @@ plugin 'cocoapods-keys', {
...
+  pod 'InterAppCommunication'
+
+  # Artsy Spec repo stuff
+  pod 'Artsy+UILabels', :head
+  pod 'Artsy+UIColors'
+  pod 'Artsy-UIButtons', '~> 1.4'
+  pod 'UIView+BooleanAnimations'
+
+  if %w(orta ash artsy laura eloy sarahscott).include?(ENV['USER']) || ENV['CI'] == 'true'
+      pod 'Artsy+UIFonts', :git => "https://github.com/artsy/Artsy-UIFonts", :branch => "old_fonts_new_lib"
```
Comment (orta) [inline Podfile]: Created a new branch that has the frameworks type of library, but no changes to the font kerning as seen in later releases.

File: Podfile [hunk 4]
```diff
@@ -25,84 +29,92 @@ plugin 'cocoapods-keys', {
...
+  else
+    pod 'Artsy+OSSUIFonts'
+  end
+
+  # Auth
+  pod 'FBSDKCoreKit', '~> 4.2'
+  pod 'FBSDKLoginKit', '~> 4.2'
+
+  pod 'AFOAuth1Client', :git => "https://github.com/orta/AFOAuth1Client", :branch => "patch-1"
```
Comment (orta) [inline Podfile]: https://github.com/AFNetworking/AFOAuth1Client/pull/113 - don't see this PR going anywhere TBH

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -1,15 +0,0 @@
-### Troubleshooting
```
Comment (orta) [inline docs/troubleshooting.md]: old old

File: docs/certs.md [hunk 6]
```diff
@@ -5,8 +5,6 @@
 * Provisioning Profile: `Artsy Beta Enterprise Provisioning Profile`
 * Certificate: `iPhone Distribution: ART SY INC`, found in the Artsy Engineering 1Password vault. (For now, this is the Feb 25, 2018 one.)
 
-##### Orta Therox - Wed 4 Sep 2013
```
Comment (orta) [inline docs/certs.md]: all documentation is obviously me, so no need to include

**Comment by orta** [comment]
Giphy

**Comment by orta** [comment]
ready for review

**Comment by alloy** [comment]
> This has been re-implmented with the new Facebook API, so it will need some manual testing.

Has this been done?

And what about on :iphone: ?

**Comment by alloy** [comment]
Otherwise the patch looks good to me :+1:

**Comment by orta** [comment]
> And what about on :iphone: ?

Doing it on an iPad ATM in-between eidolon travis runs. :+1:

**Comment by orta** [comment]
Confirmed. I can log in to my account via Facebook when running on an iPad in store config :+1:

**Comment by orta** [comment]
Alright, I'm doing it. 

MRW I want to hop on an old bandwagon but I don't have any fancy software

**Comment by alloy** [comment]
Awesome :+1:

**Pull request merged**